### PR TITLE
Optimize sidebar navigation

### PR DESCRIPTION
**Pull request by chenxsan**
The current menus in sidebar are flat, take https://webpack.js.org/guides/development/ for example:

image

Which is actually misleading. There are four heading 2 with three heading 3. This pull request adds hierarchy to the sidebar:

image

<del>Currently `remark-extract-anchors` would resolve to my own git repo https://github.com/chenxsan/remark-extract-anchors.git#support-levels-option, I've already submitted a pull request [here](https://github.com/montogeek/remark-extract-anchors/pull/4), will update it when the pull request get merged by @montogeek </del>

### CODE REVIEW

**Comment by chenxsan** [comment]
Just setup a preview here https://webpack-acb0ezj36.now.sh/contribute/

**Comment by montogeek** [comment]
Thanks! I just published 1.1.0 with your changes https://www.npmjs.com/package/remark-extract-anchors/v/1.1.0

**Comment by anikethsaha** [comment]
Nice work 🎉  , the new sidebar really looks fresh 👍

File: yarn.lock [hunk 1]
```diff
@@ -2,26 +2,31 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
```
Comment (EugeneHlushko) [inline yarn.lock]: So many changes here, were you on node 10 or greater?

Comment (chenxsan) [reply yarn.lock]: I'm on node v10.19.0, should I downgrade to v8 to update `yarn.lock`? As far as I know node v8 was moved into maintenance mode https://nodejs.org/en/about/releases/

Comment (EugeneHlushko) [reply yarn.lock]: no, 10 is good

**Comment by EugeneHlushko** [comment]
All right, i couldnt find any potential issues, lets release and look out for issues

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**